swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: start
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: stop
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zrevrangebylex/:id/:min/:max':
+    get:
+      description: 'Controller: ms. Action: zrevrangebylex.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: min
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: max
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zrevrangebyscore/:id/:max/:min':
+    get:
+      description: 'Controller: ms. Action: zrevrangebyscore.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: max
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: min
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zrevrank/:id/:member':
+    get:
+      description: 'Controller: ms. Action: zrevrank.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: member
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zscore/:id/:member':
+    get:
+      description: 'Controller: ms. Action: zscore.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: member
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /ms/_zunionstore:
+    post:
+      description: 'Controller: ms. Action: zunionstore.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/profiles/:id':
```

Comment:
missing also methods for route '/profiles/:id':
*  "get" => security/getProfile
* "post" => security/updateProfile
* "delete" => security/deleteProfile